src/Tile/Tile.js

```diff
@@ -47,13 +49,19 @@ Tile.propTypes = {
     colorAccent: PropTypes.number,
     columnSpan: CustomPropTypes.range(1, 6),
     disabled: PropTypes.bool,
+    productTile: PropTypes.bool,
```

Comment:
I think this component should clone children and pass the top-level `productTile` prop to each child so the prop only needs to be added at the top-level component. (see related comments)